### PR TITLE
feat: Add entity search cannot find footer

### PR DESCRIPTION
**Pull request by yeahfro**
## Description of change
Adds the "cannot find" footer for when the user cannot find the entity they are looking for. The footer guides them through correcting the problem.

### Before
Screenshot 2019-08-15 at 22 05 19

### After
Screenshot 2019-08-15 at 22 05 05

### CODE REVIEW

File: src/entity-search/EntitySearch.jsx [hunk 1]
```diff
@@ -29,6 +34,7 @@ EntitySearch.propTypes = {
     meta: PropTypes.object.isRequired,
   })).isRequired,
   onEntitySearch: PropTypes.func.isRequired,
+  cannotFind: PropTypes.object.isRequired,
```
Comment (rafenden) [inline src/entity-search/EntitySearch.jsx]: Could we please use `PropTypes.shape` instead?

https://stackoverflow.com/a/45764828/3042543

File: src/entity-search/EntitySearch.test.jsx [hunk 2]
```diff
@@ -12,7 +13,7 @@ const API_ENDPOINT = 'http://localhost:8000/v4/dnb/company-search'
 const mock = new MockAdapter(axios)
 mock.onPost(API_ENDPOINT).reply(200, companySearchFixture)
 
-function tick() {
+function flushPromises() {
```
Comment (rafenden) [inline src/entity-search/EntitySearch.test.jsx]: Alternatively `setImmediate` might be used.

File: src/entity-search/EntitySearch.stories.jsx [hunk 3]
```diff
@@ -28,6 +28,18 @@ storiesOf('EntitySearch', module)
...
               getEntities={dnbCompanySearchDataProvider(apiEndpoint)}
+              cannotFind={{
+                summary: 'I cannot find the company I am looking for',
+                actions: [
+                  'Check the country selected is correct',
+                  'Check for spelling errors in the company name',
+                  'Remove or add Ltd or Limited to your search',
+                ],
+                link: {
+                  url: 'http://blah.com',
```
[the remark this answers is not in the document]
Comment (yeahfro) [reply src/entity-search/EntitySearch.stories.jsx]: Yeah good idea, I was thinking similar

File: src/entity-search/EntitySearch.jsx (hunk 1 above)
Comment (yeahfro) [reply src/entity-search/EntitySearch.jsx]: Yes this is a mistake 😂

File: src/entity-search/EntitySearch.test.jsx (hunk 2 above)
Comment (yeahfro) [reply src/entity-search/EntitySearch.test.jsx]: I tried a few combinations and the tests break.. if you know something that works better then please provide a working code example.

**Pull request merged**